src/module.c

```diff
@@ -4881,6 +4881,21 @@ void RM_ThreadSafeContextLock(RedisModuleCtx *ctx) {
     moduleAcquireGIL();
 }
 
+/* Similar to RM_ThreadSafeContextLock but this function
+ * would not block if the server lock is already acquired
+ * if successful (lock acquired) REDISMODULE_OK is returned
+ * otherwise REDISMODULE_ERR is returned. */
```

Comment:
```suggestion
/* Similar to RM_ThreadSafeContextLock but this function
 * would not block if the server lock is already acquired.
 *
 * If successful (lock acquired) REDISMODULE_OK is returned,
 * otherwise REDISMODULE_ERR is returned and errno is set
 * accordingly. */
```